Gemfile

```diff
@@ -52,10 +56,11 @@ end
 
 group :test do
```

Comment:
For this project, instead of using `minitest` let us use `Rspec`. In relation to this we also need to install `factory_bot_rails` and `capybara`.

Reply:
:thinking: Is there a particular reason we choosing`Rspec` over `minitest` ?

Reply:
Great question. Well frankly speaking for this project size it doesn't matter which suite you use.

However, we in the TSE team (@dipto0321, alvaro and emily) have previously concluded we'd want you guys to be well versed with RSpec:

1. It's the more commonly used test suite in the industry
2. RSpec is more robust and test data using factories is less brittle than minitest.

Since we are being trained in Microverse to be a well-rounded and highly skilled engineers it will profit you well to follow that suggestion :)

Reply:
Thanks for the explanation. I understand :+1: